### PR TITLE
feat: summoner (challenges) page improvements

### PR DESCRIPTION
**Pull request by GinMitch**
fixed existing filtration; added more filtration options; added display mode toggle

### CODE REVIEW

File: assets/title.svg.tsx
Comment (DarkIntaqt) [inline assets/title.svg.tsx:1]: this is still the same file right? Just formatted

Comment (GinMitch) [reply assets/title.svg.tsx:1]: yes

**Comment by DarkIntaqt** [comment]
I think we can ignore the "checks" here, that's nothing major.

**Review by DarkIntaqt — APPROVED** [review]
LGTM

**Pull request merged**